src/views/blog/BlogIndex.vue

```diff
@@ -1,40 +1,190 @@
 <template>
   <div>
-    <div id="blogPostList">
-      <blog-post-component
-      v-for="post in currentBlogPosts"
-      :key="post.id"
-      :post="post">
-      </blog-post-component>
+    <div class="featured-section">
+      <div class="featured-image">
+        <img :src="featuredPost.coverImageURL">
+      </div>
+      <div class="featured-content">
+        <div class="tags-container">
+          <div v-for="tag in featuredPost.tags" class="tag">
+            {{tag}}
+          </div>
+        </div>
+        <div class="title">
+          Featured: {{featuredPost.title}}
+        </div>
+        <div class="half-divider"/>
+        <div class="sub-title">
+          {{featuredPost.subTitle}}
+        </div>
+        <div class="author-info">
+          {{featuredPost.author.firstName}} {{featuredPost.author.lastName}} {{featuredPost.createdAt}}
+        </div>
+        <div class="sample-description">
+          {{truncateFeature(featuredPost.body)}}
+        </div>
+      </div>
+    </div>
+    <div :class="getClassName(post)" v-for="post in otherPosts" v-bind:key="post.id">
```

Comment:
rest of posts. Also getClassName changes whether or not we are using a border on the bottom. Essentially removes the border on the bottom from the last post on the list. Once we load more posts this should dynamically change to add a border to the current last post and remove it from the new last post in the future